Shared/Samples/Edit feature attachments/EditFeatureAttachmentsView.swift

```diff
@@ -0,0 +1,232 @@
+                                        }
+                                        try await model.add(
+                                            name: "Attachment",
+                                            type: "png",
+                                            dataElement: pngData
+                                        )
+                                    } catch {
+                                        self.error = error
+                                    }
+                                }
+                            })
+                        } else {
+                            let attachment = model.attachments[index]
+                            AttachmentView(attachment: attachment, onDelete: { attachment in
+                                Task {
+                                    do {
+                                        try await model.delete(attachment: attachment)
+                                    } catch {
+                                        self.error = error
+                                    }
+                                }
+                            })
+                        }
+                    }
+                }
+            }
+            .errorAlert(presentingError: $error)
+        }
+    }
+}
+
+private extension EditFeatureAttachmentsView {
+    // MARK: - CalloutView
+    
+    struct CalloutView: View {
+        /// The data model for the sample.
+        @ObservedObject var model: Model
+        
+        var body: some View {
+            VStack(alignment: .leading, spacing: 3) {
+                Text(model.calloutText)
+                    .font(.callout)
+                    .multilineTextAlignment(.leading)
+                Text(model.calloutDetailText)
+                    .font(.footnote)
+                    .multilineTextAlignment(.leading)
+            }
+        }
+    }
+}
+
+private extension EditFeatureAttachmentsView {
+    // MARK: - AttachmentView
+    
+    struct AttachmentView: View {
+        // The attachment that is being displayed.
+        let attachment: Attachment
+        // The closure called when the delete button is tapped.
+        let onDelete: ((Attachment) -> Void)
+        // The image in the attachment.
+        @State private var image: Image?
+        
+        var body: some View {
+            HStack {
+                Text("\(attachment.name)")
+                    .font(.title3)
+                Spacer()
+                Button {
+                    Task {
+                        let result = try await attachment.data
+                        if let uiImage = UIImage(data: result) {
+                            image = Image(uiImage: uiImage)
+                        }
+                    }
+                } label: {
+                    if let image = image {
+                        image
+                    } else {
+                        Image(systemName: "icloud.and.arrow.down.fill")
```

Comment:
```suggestion
                        Image(systemName: "arrow.down.circle.fill")
```
A download symbol should be used here instead. iCloud symbols may only be used to refer to Apple's iCloud service.